Sources/SpeziSecureStorage/Credentials/Credential.swift

```diff
@@ -6,9 +6,12 @@
 // SPDX-License-Identifier: MIT
 //
 
+/// A user's credential containing username and password.
+@available(*, deprecated, renamed: "Credential")
+public typealias Credentials = Credential
 
-/// A pair of username and password credentials.
-public struct Credentials: Equatable, Identifiable {
+/// A user's credential containing username and password.
+public struct Credential: Equatable, Identifiable {
     /// The username.
     public var username: String
     /// The password.
```

Comment:
nevermind, changed it and deprecated SecureStorage as a whole. We now don't even have SecureStorage on Android anymore, since it isn't needed and just builds a Façade that isn't really necessary.

Reply:
I think that would make sense to ensure that we don't have to take a breaking change for now.

Reply:
But soft-deprecation is fine, right? Like I still kept the implementation of SecureStorage on iOS, but marked the whole class as deprecated. So, one can still use it as-is, but internally we are pretty much just calling the new functions instead.

Credential now has a `server` property, but if you use the deprecated functions, we internally only use the function parameter and ignore what was set in the Credential struct.

Reply:
Exactly; that would be ideal. Once we tag a new major version we can remove all these deprecated function implementations.

Sounds like a good plan!